src/dbc.cpp

```diff
@@ -158,6 +276,21 @@ void DbcParser::parse_dbc_messages(const std::vector<std::string>& lines) {
 
 			Signal sig(name, is_multiplexed, start_bit, size, is_bigendian, is_signed, factor, offset, min, max, unit, receivers);
 			messages.back().appendSignal(sig);
+			continue;
+		}
+
+		if (parseVal(line, obj)) {
+			sv.push_back(obj);
+			continue;
+		}
+	}
+
+	for (const auto& obj : sv) {
+		for (auto& msg : messages) {
+			if (msg.id() == obj.can_id) {
+				msg.addValueDescription(obj.signal_name, obj.vd);

```

Comment:
I see now that this is done through a function on the message. I see why it is done this way now I think.